### PR TITLE
Minor update of Android NDK

### PR DESCRIPTION
**Pull request by mikuru396**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Minor version update of Android NDK, please go to https://developer.android.com/ndk/downloads/index.html for revision history.

### CODE REVIEW

File: Formula/android-ndk.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class AndroidNdk < Formula
-  desc "Android native-code language toolset"
+  desc "Android native-code language toolset (Native Development Kit)"
   homepage "https://developer.android.com/ndk/index.html"
-  url "https://dl.google.com/android/repository/android-ndk-r12-darwin-x86_64.zip"
-  version "r12"
-  sha256 "fe10b64a65f3e818b852862975ea7b5896295cd1301d1acdbeb1294d9592fb19"
+  url "http://dl.google.com/android/repository/android-ndk-r12b-darwin-x86_64.zip"
```
Comment (DomT4) [inline Formula/android-ndk.rb]: Please don't downgrade to `http://` here. Should still be able to use `https://`.

File: Formula/android-ndk.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+  version "r12b"
```
Comment (DomT4) [inline Formula/android-ndk.rb]: Apparently safe to scrap this line now.

**Comment by dunn** [comment]
🔔

**Comment by eha11** [comment]
Here's the sha256 if need it.  `2bdef9143a2c7680fcb7c9fd54fe85013d591f106aea43831eba5e13e10db77e`
Not sure if that is why this failed.

**Comment by DomT4** [comment]
Closing in favour of https://github.com/Homebrew/homebrew-core/pull/4045 since this seems to have gone dead, but thank you for the submission to Homebrew @mikuru396 ❤️.

**Pull request closed without merging**